packages/suite/src/actions/wallet/graphActions.ts

```diff
@@ -159,15 +171,30 @@ export const updateGraphData =
             return;
         }
 
-        dispatch({
-            type: AGGREGATED_GRAPH_START,
-        });
-        const promises = filteredAccounts.map(
-            a => dispatch(fetchAccountGraphData(a)), // fetch for all range
-        );
-        await Promise.all(promises);
+        try {
+            dispatch({
+                type: AGGREGATED_GRAPH_START,
+            });
+            const promises = filteredAccounts.map(
+                a => dispatch(fetchAccountGraphData(a, { abortSignal })), // fetch for all range
+            );
+            await Promise.all(promises);
 
-        dispatch({
-            type: AGGREGATED_GRAPH_SUCCESS,
-        });
-    };
+            abortSignal?.throwIfAborted();
+
+            dispatch({
+                type: AGGREGATED_GRAPH_SUCCESS,
+            });
+
+            return;
+        } catch (error) {
+            if (error.name.toLowerCase().includes('aborterror')) {
```

Comment:
nit: a more idiomatic way
```suggestion
            if (error.name === 'AbortError') {
```

Reply:
I am not yet sure about what does it throws... So I'll check

Reply:
yeah it's this, I was dealing with it recently.
```
a = new AbortController(); a.abort(); try{a.signal.throwIfAborted() }catch(e){console.log(e.name)}
```